feedback/20220614.md

```diff
@@ -0,0 +1,35 @@
+# First Signals from Dev Community
+
+> see [communication-management/issues/111](https://github.com/blindnet-io/communication-management/issues/111)
+
+## Direct Feedback
+
+### Big Corps Still Need Guidance
+
+Even when a big company _has_ a security/privacy expertise team, some other teams might not have access to it, and can still therefore benefits from using our tools.
+
+### Identity management
+
+Identity and privacy are strongly linked. Especially in a context where cross-referring data between a wide variety of context allow inferring very personal information.
+
+The "v1" approach (i.e. identifying a submitter only by a unique id linked to one data collection) _does_ make a lot of sense in this context.
+
+However, there are still two issues with this approach:
+
+1. Privateform relies heavily on email address for RGPD rights management, which can't be considered a secured solution, as it can be easily stolen
+2. no real solution is provided to help submitters to easily manage the private information they submitted
+
+Users (and especially developers) focus a lot on solution easing identity management, as ease of use stay, for now, more important than actual trust.
```

Comment:
it would be usefull to have your oppinion about [what is proposed in v2](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#decentralised-identity-of-data-subjects) and your feeling on whether this would work for the developers.